.gitignore

```diff
@@ -1,7 +1,7 @@
-/rust/target/
+target
```

Comment:
This would result in ignoring 'target' directories outside of the 'rust' directory. That means that this setting would ignore things that the dev does not know about, but would expected to be told about by 'git status'.

Reply:
After setting up direnv, I'm getting artifacts from `just build` in `./target`

Reply:
That should not be. Send repro instructions?

Reply:
I guess those were repro instructions, actually. I don't get that directory. It is probably leftover from when you switched that directory from one that built in the top level to one that built in the 'rust' subdirectory.

Reply:
After seeing this phenomenon in multiple terminals on various branches yesterday, it appears to no longer be a problem for me. I guess, the future is bright.